frontend/src/app/spot/styles/sass/components/action-bar.sass

```diff
@@ -1,41 +1,29 @@
 .spot-action-bar
   display: grid
   grid-template-columns: 1fr
-  padding: $spot-spacing-1
+  padding: 0 $spot-spacing-1 $spot-spacing-0_75 0
   background-color: $spot-color-basic-gray-6
 
+  &__transparent
```

Comment:
This should be a single underscore

Reply:
wait, didn't you tell Andreas just last week, that it should have been double underscore for the modifiers? If we go BEM?

Reply:
We probably misunderstood eachother then. It's double dash for elements, single underscore for modifiers. It's only convention in the end, maybe he took it from the [official docs](https://en.bem.info/methodology/quick-start/)

Reply:
Ah, I see the difference. We do not really use the element notation.